### DISCUSSION TITLE
Daisy Chain Tasks

### DISCUSSION THREAD

**Discussion by scottcantwell** [dahall/TaskScheduler#975] [Q&A]
Is there a way to setup so that tasks can run after one completes?

**Comment by dahall** [comment]
The only way I know to do this is if each task calls a program that posts a completion event to the system Event Log. Then, you could use an EventTrigger to watch for those events to start. See the Microsoft detail on [IEventTrigger](https://learn.microsoft.com/en-us/windows/win32/api/taskschd/nn-taskschd-ieventtrigger) to understand how Event Triggers work.
